cli/src/lib/cacheRepoUtils.js

```diff
@@ -105,7 +105,6 @@ export async function ensureCacheRepo(
         }
 
         if (lastUpdated + cacheRepoExpiry < Date.now()) {
-          console.log('• rebasing flow-typed cache...');
```

Comment:
Why remove this? Because we have options like skip cache users may want to see this

Reply:
Good catch, that was removed by accident